simplesat/transaction.py

```diff
@@ -29,16 +30,20 @@ class RemoveOperation(Operation):
 class Transaction(object):
 
     def __init__(self, pool, decisions, installed_map):
-        self.operations = []
-
+        self.operations = self._safe_operations(pool, decisions, installed_map)
+        self.pretty_operations = []
```

Comment:
I think we should remove the `pretty_operations` computation (and all the related code in `_compute_transaction`), but in this case, the `Transaction` object should take care of generating nice update operations from the `operations` attribute.